addon/components/frost-autocomplete.js

```diff
@@ -0,0 +1,125 @@
+import Ember from 'ember'
+import computed, {readOnly} from 'ember-computed-decorators'
+import {PropTypes} from 'ember-prop-types'
+import layout from '../templates/components/frost-autocomplete'
+import {keyCodes} from '../utils'
+import FrostSelect from './frost-select'
+
+const {$, get, isEmpty, run} = Ember
+const {DOWN_ARROW, UP_ARROW} = keyCodes
+
+export default FrostSelect.extend({
+  layout,
+  classNames: 'frost-select',
```

Comment:
This doesn't come from `FrostSelect` already?  Did you mean for this to be a different value?